src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -173,6 +173,14 @@ public function section($key, $template=null)
 		if ($template === null)
 		{
 			$template = 'block_section';
+			
+			foreach ($this->positions as $position)
+			{
+				if (isset($position[$key]))
```

Comment:
Normally the [$template] value should exist for sure, because in the dataset from the section wizard always save title, id , template and position. 
But now we are safe.